Source/WebCore/platform/SharedMemory.h

```diff
@@ -115,6 +115,20 @@ class SharedMemory : public ThreadSafeRefCounted<SharedMemory> {
     std::span<const uint8_t> span() const { return { static_cast<const uint8_t*>(m_data), m_size }; }
     std::span<uint8_t> mutableSpan() const { return { static_cast<uint8_t*>(m_data), m_size }; }
 
+    std::span<const uint8_t> subspan(size_t offset, size_t count) const
```

Comment:
This goes against the idea of std::span.
The idea is to obtain the span and then limit it with subspan()

Reply:
🆗 I just thought that `span(...).subspan()` is ugly